test/Microsoft.AspNetCore.Server.HttpSys.FunctionalTests/Listener/ServerTests.cs

```diff
@@ -245,6 +245,23 @@ public class ServerTests
             }
         }
 
+        [ConditionalFact]
+        public async Task Server_SetRejectionVerbosityLevel_Success()
+        {
+            string address;
+            using (var server = Utilities.CreateHttpServer(out address))
+            {
+                server.Options.Http503ResponseVerbosityLevel = Http503ResponseVerbosityLevel.Limited;
+                var responseTask = SendRequestAsync(address);
+
+                var context = await server.AcceptAsync(Utilities.DefaultTimeout);
+                context.Dispose();
+
+                var response = await responseTask;
+                Assert.Equal(string.Empty, response);
```

Comment:
This doesn't really test the scenario does it? This isn't a 503 response. Can you cut the connection limit or similar to force it?

Reply:
Trying to hit throttle conditions for unit tests is a recipe for flaky tests. Following the same approach used for RequestQueueLimit here - checking that it is possible to set the value in http.sys. I don't think testing that http.sys handles the setting correctly is necessary here (or worth the hassle of flakiness when trying to hit throttling conditions in http.sys) - if http.sys breaks that behavior they'll have test coverage for it themselves.

Reply:
What if you didn't accept any requests and set the queue limit to 1 or 0? Or similarly set the connection limit to 1 or 0?